### PR TITLE
Putting "Additional Input Required" doesn't bar you from creating a booking #2660

### LINKED ISSUE #2660: Putting "Additional Input Required" doesn't bar you from creating a booking
Found a bug? Please fill out the sections below. 👍

### Issue Summary

when you click 'is required' for a new custom input field, you can still make a booking without filling in the custom input field

### Steps to Reproduce

1. (for example) Went to event types
2. Clicked on an event
3. Set an addtl input to be 'required'
4. Went and tried to book the link on incognito mode without entering the required information
5. Was still able to book with the 'required' info not put in

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### PR DESCRIPTION
**Pull request by gitstart**
## What does this PR do?

Fixes #2660

Loom Video: https://www.loom.com/share/baba361ae36d4c1ba6ee6e97369c09ce

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/components/booking/pages/BookingPage.tsx [hunk 1]
```diff
@@ -642,9 +642,8 @@ const BookingPage = ({
                         )}
                         {input.type === EventTypeCustomInputType.TEXTLONG && (
                           <textarea
-                            {...bookingForm.register(`customInputs.${input.id}`, {
-                              required: input.required,
```
Comment (zomars) [inline apps/web/components/booking/pages/BookingPage.tsx:646]: Why did we remove from hook form? IMO we should leave it in there as well.

**Review by zomars — CHANGES_REQUESTED** [review]
This solutions works but it will only work on the client only. We should validate on the server as well.

**Comment by zomars** [comment]
@gitstart on this PR would be optimal 🙏

**Review by zomars — CHANGES_REQUESTED** [review]
We're still missing server-side validations, please only re-request review when all comments are fully addressed. 🙏

**Comment by PeerRich** [comment]
does not solve the problem

**Pull request closed without merging**